src/js/components/BentoAppRouter.tsx

```diff
@@ -26,19 +26,31 @@ const BentoAppRouter = () => {
   const isAuthenticated = useIsAuthenticated();
 
   useEffect(() => {
-    dispatch(makeGetConfigRequest()).then(() => dispatch(getBeaconConfig()));
-    dispatch(makeGetAboutRequest());
-    dispatch(makeGetDataRequestThunk());
-    dispatch(makeGetSearchFields());
-    dispatch(makeGetProvenanceRequest());
-    dispatch(makeGetKatsuPublic());
-    dispatch(fetchKatsuData());
-    dispatch(fetchGohanData());
-    dispatch(makeGetServiceInfoRequest());
-    //TODO: Dispatch makeGetDataTypes to get the data types from service-registry
-    if (isAuthenticated) {
-      dispatch(makeGetDataTypes());
-    }
+    const fetchInitialData = async () => {
+      try {
+        await dispatch(makeGetConfigRequest());
+        await dispatch(getBeaconConfig());
+        await dispatch(makeGetAboutRequest());
+
+        await Promise.all([dispatch(makeGetDataRequestThunk()), dispatch(makeGetSearchFields())]);
+
+        dispatch(populateClickable());
+
+        await dispatch(makeGetProvenanceRequest());
+        await dispatch(makeGetKatsuPublic());
+        await dispatch(fetchKatsuData());
+        await dispatch(fetchGohanData());
+        await dispatch(makeGetServiceInfoRequest());
```

Comment:
Lots of awaits, but are they all needed? The previous code only has a single action going through `then`, so why so many awaits now?